app/views/admin/republishing/confirm.html.erb

```diff
@@ -0,0 +1,16 @@
+<% content_for :page_title, "Are you sure you want to republish \"#{@document_title}\"?" %>
+<% content_for :title, "Are you sure you want to republish \"#{@document_title}\"?" %>
```

Comment:
For the one that's the document title not the heading, should we simplify to `"Republish \"#{@document_title}\""`? I can't remember if using different content for the document title and `h1` is bad practice

Reply:
We've changed the title here.